src/content/reference/manifest.mdx

```diff
@@ -72,6 +72,8 @@ You can build all these images by running `okteto build`, or `okteto build xxx`
 
 > if a `<service>` name includes the symbol `-`, in the corresponding `OKTETO_BUILD_<service>` environment variables, the symbol `-` will be replaced by `_`.  For example, if `<service>` is `name-with-hyphen`, the environment variable names will start with `$OKTETO_BUILD_NAME_WITH_HYPHEN_`.
 
+> Okteto autoinjects `OKTETO_NAMESPACE` and `OKTETO_CONTEXT` on every build as build args.
```

Comment:
@jLopezbarb what's `OKTETO_CONTEXT`?

Reply:
Okteto context is the context name on the `context.json`. Should we include `OKTETO_URL` instead?

Reply:
I don't remember why we implemented this, but the name of the context don't look very useful to me